docs/website/mmp-errors.md

```diff
@@ -277,7 +277,21 @@ See the [equivalent mtouch warning](mtouch-errors.md#MT5218).
 ### <a name="MM5310">MM5310: install_name_tool failed with an error code '{0}'. Check build log for details.
 
 <!-- MM6xxx: mmp internal tools -->
-<!-- MM7xxx: reserved -->
+
+# MM7xxx: MSBuild error messages
+
+<!--
+ MM7xxx msbuild errors
+  -->
+
+## <a name="MM7068"/>MM7068: Cannot expand $(TeamIdentifierPrefix) in Entitlements.plist without a provisioning profile.
+
+The $(TeamIdentifierPrefix) variable used in Entitlements.plist is expanded from the value defined in the Provisioning Profile, so without a Provisioning Profile set in the Project Options, the variable cannot be expanded.
+
+## <a name="MM7068"/>MM7069: Cannot expand $(AppIdentifierPrefix) in Entitlements.plist without a provisioning profile.
```

Comment:
You're not logging MM7069, only MT7069. Also copy and paste mistake the <a> should be `MM7069`.